locales/en.json

```diff
@@ -1,5 +1,8 @@
 {
-	"This service is to claim tax benefits online.": "This service is to claim tax benefits online.",
-	"Claim tax benefits": "Claim tax benefits",
-	"Please correct the errors on the page": "Please correct the errors on the page"
-}
\ No newline at end of file
+  "This service is to claim tax benefits online.": "This service is to claim tax benefits online.",
+  "Claim tax benefits": "Claim tax benefits",
+  "Please correct the errors on the page": "Please correct the errors on the page",
+  "errors.login.length": "Access code must be 8 characters",
```

Comment:
Why are we having some strings with identifiers and others without?

Reply:
The validation messages were being put directly in the schemas, which is a third place, so I didn't want to keep them there.

The other two places are:

1. directly in the template
2. in the i18n locales json files

Since error messages are conceptually related, they're an easy candidate to pick off, so that we group them together in the i18n files and update them all at once down the road.